### ISSUE TITLE
Problems in the process of reproducing the code

### ISSUE THREAD

**Issue by ChenJing-825** [jind11/TextFooler#44]
Thank you for open-sourcing the code! 
In the process of reproducing, the following problems appeared:
Error(s) in loading state_dict for BertForSequenceClassification:
        size mismatch for classifier.weight: copying a param with shape torch.Size([4, 768]) from checkpoint, the shape in current model is torch.Size([2, 768]).
        size mismatch for classifier.bias: copying a param with shape torch.Size([4]) from checkpoint, the shape in current model is torch.Size([2]).

I haven't solved it, can you help me? Thanks!

**Comment by jind11** [comment]
Hi, this error came out when you were loading the model parameters. Could you re-check the number of labels in the model configuration (currently you are using 2)? It should match the model parameters, which is 4 for the number of labels.
